assets/js/dist/fade.js

```diff
@@ -1,40 +1,38 @@
 "use strict";
 
-$(document).ready(function () {
-  var mobileOffset = 270; // Start with only work section showing
+var mobileOffset = 270; // Start with nav-work button highlighted and only work section showing
 
-  $('#nav-work').removeClass('default-btn').addClass('current-btn');
-  $('#work').addClass('display');
-  var elements = document.getElementsByClassName('nav-btn');
-  Array.from(elements).forEach(function (el) {
-    el.addEventListener('click', function (e) {
-      e.preventDefault();
-      $('li').removeClass('current-btn');
-      $("#".concat(e.target.id)).addClass('current-btn');
-      $('#fade-wrapper').fadeOut(500, function () {
-        $('section').removeClass('display');
-        var sectionId = e.target.id.slice(4);
-        $("#".concat(sectionId)).addClass('display');
+document.getElementById('nav-work').classList.add('current-btn');
+document.getElementById('work').classList.add('display');
+var navElements = document.getElementsByClassName('nav-btn');
+Array.from(navElements).forEach(function (navElement) {
+  navElement.addEventListener('click', function (e) {
+    e.preventDefault();
+    Array.from(document.getElementsByTagName('li')).forEach(function (el) {
+      return el.classList.remove('current-btn');
```

Comment:
Why do you have an explicit return here? The return value from `forEach` is ignored, so it seems odd to be going out of your way to return this value.

Reply:
In my es6 I think I was trying to be slick doing it like this in one line
`.forEach(el =>  el.classList.remove('current-btn'));`
without stopping to realize that would be an explicit return.
If I want to do it in one line like this, does that work? 
`.forEach((el) => { el.classList.remove('current-btn') });`

Reply:
Oh, I totally forgot that I was reviewing the generated code (GitHub annoyingly won't let me add inline comments to code which did not change in the PR).

Yeah, technically the one with `{}` would be more correct, but I think it's fine to just use the arrow in this case and not worry about the extraneous return.